app.js

```diff
@@ -0,0 +1,22 @@
+// app.js
+// set up ======================================================================
+// get all the tools we need
+var config = require('./config');
+var morgan = require('morgan');
+var path = require('path');
+var express = require('express');
+var app = express.createServer();
+app.listen(config.port);
```

Comment:
I cannot do this given the special configuration of my app.
Listening need to be done earlier.